.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -0,0 +1,224 @@
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Parse VULNERABILITY_BUNDLE_VERSION
+      id: set-versions
+      run: |
+        versions_output=$(./.github/workflows/scripts/scanner-get-released-tags.sh)
+        echo "versions=$versions_output" >> "$GITHUB_OUTPUT"
+
+  prepare-environment:
+    needs: parse-versions
+    runs-on: ubuntu-latest
+    outputs:
+      manual_url: ${{ steps.set-manual-url.outputs.manual_url }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Download NVD
+      run: |
+        set -eu
+        since_time=$(date -u -d '24 hours ago' '+%a, %d %b %Y %H:%M:%S GMT')
+          NVD_BUNDLE_TYPE=${{ github.event.inputs.job || 'nvd-api' }}
+          case "$NVD_BUNDLE_TYPE" in
+              nvd-api)
+                  nvd_file=nvd-api.zip
+                  ;;
+              nvd-feeds)
+                  nvd_file=nvd-feeds.zip
+                  ;;
+              ,*)
+                  echo >&2 "error: invalid NVD bundle type '$NVD_BUNDLE_TYPE'"
+                  exit 1
+          esac
+          url="https://definitions.stackrox.io/v4/nvd/$nvd_file"
+
+        code=$(curl \
+            -o nvd.zip \
+            -w "%{http_code}" \
+            -H "If-Modified-Since: $since_time" \
+            "$url")
+
+        echo "code: $code"
+        echo "$code" | grep -q 200
+
+    - name: Set MANUAL_URL
+      id: set-manual-url
+      run: |
+        if [ "${{ github.event_name }}" == "schedule" ]; then
+          # Scheduled workflows should use the master branch
+          echo "manual_url=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        else
+          # Determine the SHA to use
+          if [ -z "${{ github.event.pull_request.head.sha }}" ]; then
+            # Fallback to the current ref or SHA if it's not a pull request event
+            sha=${{ github.sha }}
+          else
+            sha=${{ github.event.pull_request.head.sha }}
+          fi
+
+          echo "manual_url=https://raw.githubusercontent.com/stackrox/stackrox/$sha/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        fi
+
+    - uses: ./.github/actions/upload-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+        if-no-files-found: error
+
+  build-updater:
+    needs:
+    - parse-versions
+    - prepare-environment
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        include: ${{ fromJson(needs.parse-versions.outputs.versions) }}
+    env:
+      ROX_BUNDLE_VERSION: ${{ matrix.version }}
+      ROX_PRODUCT_TAG: ${{ matrix.tag }}
```

Comment:
Can we rename these to:

```suggestion
      BUNDLE_VERSION: ${{ matrix.version }}
      UPDATER_GIT_REF: ${{ matrix.tag }}
```

Also change the matrix from `matrix.tag` to `matrix.git_ref`?

(OK to use similar names, these are example to communicate what I mean.)